os/board/rtl8730e/src/component/soc/amebad2/fwlib/include/ameba_mipi_display.h

```diff
@@ -0,0 +1,129 @@
+#ifndef MIPI_DISPLAY_H
```

Comment:
Is there any reason why you put this header file and source file here, not in app/examples?

Reply:
Dear @sunghan-chang,
This is because currently the example is extern from our driver layer ameba_mipi_show.c, so the header is placed here. As currently there is no porting layer/ framework for mipi/lcdc, the example is just calling our driver layer, this is similar to wificsi example. So once the porting layer for lcdc is ready, i will move the relevant header to app/examples.
Thank you